releasenotes/notes/appsec-8f19fc73e7c70e7c.yaml

```diff
@@ -0,0 +1,7 @@
+---
+prelude: >
+    Adding AppSec based on a simple gateway
```

Comment:
Do we want to be this public about AppSec yet? If the feature is still opt-in and not officially GA we probably don't want to be publicizing it in the release notes just yet.

Reply:
Yeah so we typically do release notes for things that are GA (documented and have public APIs available to use) so I think we can hold off on the release note.